client/src/app/components/forms/inputs/forms/docker-input-form.component.ts

```diff
@@ -0,0 +1,31 @@
+import {Component, OnInit} from "@angular/core";
+import {DockerInput} from "../types/docker-input.component";
+import {InputFromComponent} from "./input-form.component";
+
+@Component({
+    selector: 'docker-input-form',
+    directives: [DockerInput, InputFromComponent],
+    template: `
+            <input-form [primaryLabel]="'Docker image'" 
+                        [secondaryLabel]="'Docker Repository'"
+                        [inputData]="inputData"
+                        [contentComponent]="contentComponent">
+            </input-form>
+    `
+})
+export class DockerInputForm implements OnInit {
```

Comment:
We should be consistent about naming our classes. From what I've seen in the Angular tutorials, the practice is to include the class type in the class name. This class has a `@Component` decorator, so it should be called `DockerInputFormComponent`.

Granted, it is a bit wordy, but I think the intent is to prevent confusion when importing the class into a different file. If it has Component in the name, its purpose is obvious.

It's up for discussion, though, certainly.